lib/vsc/install/ci.py

```diff
@@ -58,6 +58,7 @@
 EASY_INSTALL_TOX = 'easy_install_tox'
 PY3_ONLY = 'py3_only'
 PY3_TESTS_MUST_PASS = 'py3_tests_must_pass'
+PY39_TESTS_MUST_PASS = 'py39_tests_must_pass'
```

Comment:
it's probably best to rename the previous `PY3_TESTS_MUST_PASS` to `PY36_TESTS_MUST_PASS`, or make it an alias; also in the config files

Reply:
it's only there for deprecations warning anymore. e.g. if it's set you get a deprecation warning to remove it

Reply:
I don't follow, why would we add a `py36_tests_must_pass` parameter?

Reply:
because we will start having repos that are only py39 compatible